src/slack.js

```diff
@@ -48,14 +48,22 @@ const updateSlackStatus = async ({ token, text, emoji }) => {
  * or not the other promises have resolved.
  * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all
  */
-module.exports = (isInMeeting = false, workspaces = slackWorkspaces) =>
+module.exports = (eventUserMail = '', isInMeeting = false, workspaces = slackWorkspaces) =>
```

Comment:
```suggestion
module.exports = (isInMeeting = false, workspaces = slackWorkspaces, eventUserMail = '') =>
```

Putting it at the end - as it is a new arg - will reduce the amount of failing tests (see results on [travis](https://travis-ci.com/github/natterstefan/zoom-slack-status-updater/builds/166518110#L445)).

Speaking of tests, can you add some for this feature too, please? In [both server and slack test files](https://github.com/natterstefan/zoom-slack-status-updater/blob/master/src/__tests__).

Reply:
Changed the order of the argument and fixed some tests. But I am not able to write new ones and fix some of the existing. These are my first lines of JavaScript, so it takes me a lot of time to dig into the whole testing setup.

Reply:
Hi @ruben-burr,

thanks for your feedback. I just took care of the open tasks and pushed it here: https://github.com/natterstefan/zoom-slack-status-updater/pull/2/commits/624ae96ac4154919b85707d4475ca5b1660171ef.